metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj

```diff
@@ -423,39 +420,50 @@
             :errors ["Finding concept type [tag-association] with parameters [provider-id] is not supported."]}
            (util/find-concepts :tag-association {:provider-id "REG_PROV"})))))
 
-;; Failing in external Oracle
```

Comment:
Did you test with external db? Is this no longer failing?

Reply:
Ah! Looking at the history for CMR-4335, and you updated it yesterday to "needs-testing" -- fantastic!

Reply:
yes. It works with external db now. I am going to move CMR-4335 to done once this PR is merged.